bundles/org.openhab.binding.wundergroundupdatereceiver/README.md

```diff
@@ -0,0 +1,115 @@
+# Wunderground Update Receiver Binding
+
+Many personal weather stations are only capable of submitting measurements to the wunderground.com update site.
+
+This binding enables acting as a receiver of updates from weather stations that post measurements
+to https://rtupdate.wunderground.com/weatherstation/updateweatherstation.php. If the hostname is configurable - as on
+weather stations based on the Fine Offset Electronics WH2600-IP - this is simple, otherwise you have to set up dns
+such that it resolves the above hostname to your server, without preventing the server from resolving the proper ip if
+you want to forward the request.
+
+The server thus listens at http(s)://<your-openHAB-server>:<openHAB-port>/weatherstation/updateweatherstation.php and the weather station
+needs to be pointed at this address. If you can't configure the weather station itself to submit to an alternate hostname you would need
+to set up a dns server that resolves rtupdate.wunderground.com to the IP-address of your server and provide as dns to the weather station
+does DHCP. Make sure not to use this dns server instance for any other DHCP clients. 
+
+The request is in itself simple to parse, so by redirecting it to your openhab server you can intercept the values and use them to
+control items in your home. E.g. use measured wind-speed to close an awning or turn on the sprinkler system after some time without rain.
+This binding allows you to mix and match products from various manufacturers that otherwise have a closed system.
+
+If you wish to pass the measurements on to rtupdate.wunderground.com, you can use a simple rule that triggers on the
+wundergroundupdatereceiver:wundergroundUpdateReceiver:<channel-id>:metadata#last-query-trigger to do so. It can also be used to
+submit the same measurements to multiple weather services via multiple rules.
+
+## Supported Things
+
+Any weather station that sends weather measurement updates to the wunderground.com update URLs is supported.
+It is easiest to use with stations that have a configurable target address, but can be made to
+work with any internet-connected weather station, that gets it's dns server via DHCP or where the DNS server can be
+set. 
+
+## Discovery
+
+The binding starts listening at the above-mentioned URI as soon as it is initialized. Any request with an unregistered
+stationId is recorded and if auto-discovery is enabled appers in the inbox, otherwise can be registered when a manual
+scan is initiated. For each request parameter a channel is generated, based on a list of known parameters from
+https://support.weather.com/s/article/PWS-Upload-Protocol?language=en_US and other observed parameters from various
+weather station. If you have a weather station that submits a parameter that is unknown in the current version of the
+binding please feel free to submit an issue to have it added.
```

Comment:
```suggestion
The binding starts listening at the above-mentioned URI as soon as it is initialized.
Any request with an unregistered stationId is recorded and if auto-discovery is enabled appers in the inbox, otherwise can be registered when a manual scan is initiated.
For each request parameter a channel is generated, based on a list of known parameters from https://support.weather.com/s/article/PWS-Upload-Protocol?language=en_US and other observed parameters from various weather station.
If you have a weather station that submits a parameter that is unknown in the current version of the binding please feel free to submit an issue to have it added.
```

new line for each.